src/components/Nav/Nav.js

```diff
@@ -1,7 +1,58 @@
 import React, { Component } from 'react';
+import NavMain from './NavMain/NavMain';
+import NavCategory from './NavCategory/NavCategory';
+import SerachModal from './SerachModal/SerachModal';
 
 export default class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isCategotyButton: false,
+      isSerach: false,
```

Comment:
```suggestion
      isSearch: false,
```
저도 자주 실수하는 단어긴한데 올리고 나서 확인 꼼꼼하게 해주세요